src/main/java/racing/view/ResultView.java

```diff
@@ -1,25 +1,23 @@
-package view;
+package racing.view;
 
-import racing.CarPositions;
-import racing.RacingResultDto;
-
-import java.util.List;
+import racing.model.CarsAtTurn;
+import racing.model.RacingResultDto;
 
 public class ResultView {
 
     public static void printResult(RacingResultDto racingResultDto) {
         System.out.println("실행 결과");
-        for (CarPositions carPositions : racingResultDto.getPlayHistory().getPositions()) {
-            printTurnResult(racingResultDto.getCarNames(), carPositions);
+        for (CarsAtTurn carsAtTurn : racingResultDto.getPlayHistory().getPositions()) {
+            printTurnResult(carsAtTurn);
         }
 
         String winners = String.join(", ", racingResultDto.getWinnerNames());
         System.out.println(winners + "가 최종 우승했습니다.");
     }
 
-    private static void printTurnResult(List<String> carNames, CarPositions carPositions) {
-        for (int i=0; i<carNames.size(); i++) {
-            printCarPosition(carNames.get(i), carPositions.getCarPosition(i));
+    private static void printTurnResult(CarsAtTurn carsAtTurn) {
+        for (int i=0; i<carsAtTurn.getCarSize(); i++) {
+            printCarPosition(carsAtTurn.getCarName(i), carsAtTurn.getCarPosition(i));
         }
```

Comment:
```suggestion
        for (FixedCar car : carsAtTurn.getCarPositions()) {
            printCarPosition(car.getName(), car.getPosition());
        }
```

만들어주신 함수를 활용하면 이렇게도 가능하겠네요!
최대한 `index`를 통한 접근보다는 메소드호출(메세지 전달)을 통해 객체지향에 대한 접근방법을 고민해보아도 좋을것같네요!

Reply:
@wooobo 
iterator를 사용하게 수정하여 코드를 간소화하였습니다.